source/server/config_validation/server.cc

```diff
@@ -67,7 +67,9 @@ void ValidationInstance::initialize(Options& options,
   envoy::api::v2::Bootstrap bootstrap;
   InstanceUtil::loadBootstrapConfig(bootstrap, options.configPath(), options.v2ConfigOnly());
 
+  Config::Utility::detectTagNameConflict(bootstrap);
   tag_extractors_ = Config::Utility::createTagExtractors(bootstrap);
+  Config::Utility::createTags(bootstrap);
```

Comment:
nit: just for consistency, you can remove `tag_extractors_` from this file as well since it appears that it isn't being used.